src/lib/scale/create.tsx

```diff
@@ -258,7 +258,22 @@ function CreateProcurementCommand(props: CreateProcurementCommandProps) {
 }
 
 const create = new Command("create")
-  .description("create a procurement to purchase the desired number of GPUs.")
+  .description("Create a procurement to purchase the desired number of GPUs.")
+  .addHelpText(
+    "after",
+    `
+Examples:
+\x1b[2m# Create a new procurement for 8 GPUs\x1b[0m
+$ sf scale create -n 8
+
+\x1b[2m# Maintain 32 GPUs, but only while the price is <= $1.50/GPU/hr\x1b[0m
+$ sf scale create -n 32 -p 1.50
+
+\x1b[2m# Maintain 8 GPUs, start buying the next reservation when there's 30 minutes left\x1b[0m
+$ sf scale create -n 8 -d '30m'
```

Comment:
i feel like we should get rid of `-d` and just keep `--horizon` if possible

or we should use --horizon in the example instead of -d for clarity.

Reply:
yeah I think `-d` is confusing for sure. `-h` conflicts with `--help` and it helps with backwards compat with scripts written for the previous `sf scale` which accepted `--duration` was the rationale

Reply:
I'll update base `sf scale` docs as well